client/components/application-summary.js

```diff
@@ -176,15 +149,15 @@ class ApplicationSummary extends React.Component {
                 {
                   subsections.map(key => {
                     const subsection = section.subsections[key];
+                    let fields = Object.values(this.props.fieldsBySection[key] || []);
```

Comment:
Should be `const` - adding and removing elements of an array doesn't change the value of the `const`.
```suggestion
                    const fields = Object.values(this.props.fieldsBySection[key] || []);
```